build.gradle

```diff
@@ -13,6 +13,7 @@ buildscript {
 apply from: "$rootDir/gradle/java6-compatibility.gradle"
 apply from: "$rootDir/gradle/precommit.gradle"
 
+apply plugin: 'base'
```

Comment:
This is a good idea.

Later, we probably need to apply 'base' inside our org.shipkit.xxx plugins. Otherwise our users might get surprised by not working 'clean'.